{"ip_str": "104.238.182.217", "errors": {"VPN_error": "Timeout exceeded.\n<pexpect.pty_spawn.spawn object at 0x7f73fde002e0>\ncommand: /usr/sbin/openvpn\nargs: ['/usr/sbin/openvpn', '/home/NetPlatform/configurations/it2-ovpn-tcp.ovpn']\nbuffer (last 100 chars): b'nk local: (not bound)\\r\\nMon Oct 24 08:52:04 2022 TCP_CLIENT link remote: [AF_INET]138.199.54.196:80\\r\\n'\nbefore (last 100 chars): b'nk local: (not bound)\\r\\nMon Oct 24 08:52:04 2022 TCP_CLIENT link remote: [AF_INET]138.199.54.196:80\\r\\n'\nafter: <class 'pexpect.exceptions.TIMEOUT'>\nmatch: None\nmatch_index: None\nexitstatus: None\nflag_eof: False\npid: 8\nchild_fd: 5\nclosed: False\ntimeout: 30\ndelimiter: <class 'pexpect.exceptions.EOF'>\nlogfile: None\nlogfile_read: None\nlogfile_send: None\nmaxread: 2000\nignorecase: False\nsearchwindowsize: None\ndelaybeforesend: 0.05\ndelayafterclose: 0.1\ndelayafterterminate: 0.1\nsearcher: searcher_re:\n    0: re.compile(b'Initialization Sequence Completed')"}}